### DISCUSSION TITLE
Just release an Announcements module.

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1289] [Show and tell]
This was fun to create.  Its available from your Module Management page in your Oqtane site.  I am also running the module on my site [Studio Elf](http://studio-elf.net/mynews)

Built only using Oqtane and Blazor, no third party components.  Also I have implemented Dutch localization.

here are a few screen shots:
![Shoot1](https://user-images.githubusercontent.com/9447612/116212778-7d32df00-a745-11eb-8d30-321e5a15bb96.png)
![Shoot2](https://user-images.githubusercontent.com/9447612/116212852-8d4abe80-a745-11eb-85a1-786ee854c9db.png)
![Shoot3](https://user-images.githubusercontent.com/9447612/116212868-8fad1880-a745-11eb-8167-6c7907e686ac.png)
![Shoot4](https://user-images.githubusercontent.com/9447612/116212875-920f7280-a745-11eb-8b1e-e3307f47f3fe.png)

**Comment by hishamco** [comment]
Happy to hear that .. I think we can create a modules repository to add such samples something similar to DNN

@leigh-pointer regarding the localization did we miss something or you are using the default implementation?

**Reply by leigh-pointer** [reply]
@hishamco Localistion is the default implementation.  Works fine using standard and shared resources (in my module).

**Reply by sbwalker** [reply]
@leigh-pointer does localization work fine for your module on WebAssembly as well? I am curious because there is an issue #1238 which is still open.

**Reply by hishamco** [reply]
@sbwalker I don't think it's a crucial bug seems we miss to localize some strings, I will have a look when I have time

**Reply by leigh-pointer** [reply]
@sbwalker havent tried that but will set up a test.
